backends/arm/test/common.py

```diff
@@ -185,3 +189,41 @@ def get_target_board(compile_spec: list[CompileSpec]) -> str | None:
             elif "u85" in flags:
                 return "corstone-320"
     return None
+
+
+u55_fvp_mark = pytest.mark.skipif(
```

Comment:
Nit: Make it more intuitive
```suggestion
SkipIfNoCS300 = pytest.mark.skipif(
```

Reply:
Done